src/libraries/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -381,15 +365,11 @@ private static void TestRemove1(int cLevel, int threads, int removesPerThread)
             Assert.Equal(expectKeys.Count, dict.ToArray().Length);
         }
 
-        [Fact]
-        public static void TestRemove2()
-        {
-            TestRemove2(1);
-            TestRemove2(10);
-            TestRemove2(5000);
-        }
-
-        private static void TestRemove2(int removesPerThread)
+        [Theory]
+        [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(5000)]
+        public static void TestRemove2(int removesPerThread)
```

Comment:
These names are unfortunate, but they pre-exist your change, so don't worry about it; we can clean it up later.  I do like your change to use theories :)